docs/docs/segment-wifi.md

```diff
@@ -0,0 +1,50 @@
+---
+id: wifi
+title: WiFi
+sidebar_label: WiFi
+---
+
+## What
+
+Show details about the currently connected WiFi network.
+
+_Note: Currently only supports Windows and WSL. Pull requests for Darwin and Linux support are welcome :)_
```

Comment:
You can turn this into an [admonition](https://docusaurus.io/docs/markdown-features/admonitions):

```markdown
:::info
Note...
:::
```

Reply:
Done